modules/nf-core/purecn/coverage/main.nf

```diff
@@ -0,0 +1,73 @@
+process PURECN_COVERAGE {
+    tag "$meta.id"
+    label 'process_low'
+    stageInMode "link"
+
+    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
+    conda "bioconda::bioconductor-purecn=2.4.0 bioconda::bioconductor-txdb.hsapiens.ucsc.hg38.knowngene=3.16.0 bioconductor-txdb.hsapiens.ucsc.hg19.knowngene=3.2.2 bioconda::bioconductor-org.hs.eg.db=3.16.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-582ac26068889091d5e798347c637f8208d77a71:a29c64a63498b1ee8b192521fdf6ed3c65506994-0':
+        'quay.io/biocontainers/mulled-v2-582ac26068889091d5e798347c637f8208d77a71:a29c64a63498b1ee8b192521fdf6ed3c65506994-0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path intervals
+
+    output:
+    tuple val(meta), path("*.txt.gz")      , emit: txt
+    //Not generated when --skip-gc-norm is set
+    tuple val(meta), path("*.png")         , emit: png         , optional: true
+    tuple val(meta), path("*_loess_qc.txt"), emit: loess_qc_txt, optional: true
+    tuple val(meta), path("*_loess.txt.gz"), emit: loess_txt   , optional: true
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '2.4.0' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+
+    if (task.stageInMode != 'link') {
+        System.err.println("ERROR: purecn/coverage can not handle staging files with symlinks. Please change the stageInmode option to 'Link'")
+        System.exit(1)
+    } else {
```

Comment:
I think `copy` should also work here. Can you also add this code to the stub?

Reply:
Yes, pretty sure, I'm testing it asap

Reply:
I encountered a permission error on the 'tmp' directory while using the 'Copy' stage-in mode. Although this is specific to our cluster, to ensure reproducibility, I recommend using only the 'link' staging mode for now.

Reply:
allright!